core/src/main/kotlin/ch/ergon/dope/resolvable/clause/model/UseKeysClause.kt

```diff
@@ -13,42 +13,66 @@ import ch.ergon.dope.validtype.ValidType
 
 private const val USE_KEYS = "USE KEYS"
 
-sealed class SelectUseKeysClause(
-    private val parentClause: ISelectFromClause,
-    private val keys: TypeExpression<out ValidType>,
-) : ISelectUseKeysClause {
+class SelectUseKeysClause : ISelectUseKeysClause {
```

Comment:
The problem here is now that this is a public constructor, thus it is possible to create an empty _SelectUseKeysClause_ (and would eventually lead to a runtime exception when accessing useKeys or parentClause)
You should add the private constructor as proposed in https://github.com/ergon/dope-query-builder/pull/26

Reply:
(same for delete)